### ISSUE TITLE
Index column in GBTFITSLoad.summary()

### ISSUE THREAD

**Issue by astrofle** [GreenBankObservatory/dysh#78]
Personally, I find it confusing to have the index column shown when using `GBTFITSLoad.summary(verbose=False)`.
However I have not been able to find a way of returning a `pandas.DataFrame` without an index, only printing (e.g., `print(df.to_string(index=False))`). 
I like the ability of accessing the summary as a `pandas.DataFrame`, so just printing the summary seems like a poor solution.
Any ideas on how to tackle this are welcome.

**Comment by mpound** [comment]
fixed with new show_index keyword to summary(), default: False.
this calls df.style.hide(axis='index')

**Comment by astrofle** [comment]
When I tried this it did not print the index, but rather returned a function:

```
sdfits.summary()
   <pandas.io.formats.style.Styler at 0x7f0e78728410>
```

This is using `pandas==2.1.3`, when installing `dysh==0.2.0rc`

**Comment by astrofle** [comment]
Using:
```
if not show_index:
    print(compressed_df.to_string(index=False))
```
works in notebooks and the shell, however it does not look pretty in the notebooks. I'll leave it like this for now.

I'll leave the issue open, in case anyone wants to dig deeper, but this is not a priority.

**Comment by mpound** [comment]
Yeah, I originally had it like that but it looked ugly in notebooks.   I did not consider that it would not work in ipython.

**Comment by astrofle** [comment]
How about doing:
```Python
df.set_index("SCAN")
```
<img width="858" alt="Screenshot 2024-07-29 at 1 53 54 PM" src="https://github.com/user-attachments/assets/8e374f9c-73d4-49a3-8eed-7be6dd121c1c">

It looks a bit different, but I think it may help remove some of the confusion due to the index column of the DataFrame.
For `verbose=True` we could set the index to the `"ROW"` column (currently this column is not part of the summary, so I can post an example).

**Comment by mpound** [comment]
@astrofle Can this be closed due to #659
